spec/controllers/custom_styles_controller_spec.rb

```diff
@@ -85,7 +85,8 @@
         let(:valid) { true }
 
         it "redirects to show" do
-          expect(response).to redirect_to action: :show
+          expect(response).to be_redirect
```

Comment:
You are now only testing whether there is a redirect but not to which page. I guess it makes sense to check that as well as it could also be an error page and the test would be green

Reply:
This is still true, also for all of the following test cases. Are there any issues with doing this? 🤔